### PR TITLE
Homework10

### PR DESCRIPTION
**Pull request by yalokudoyar69**

### CODE REVIEW

File: src/homework10Part1/WorkingWithNumbers.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+public class WorkingWithNumbers {
+
+    static int[] creatingArray(int arraySize) {
+        int[] arr = new int[arraySize];
+        Random random = new Random();
+        for (int i = 0; i < arraySize; i++) {
+            arr[i] = random.nextInt(200);
+        }
+        for (int i = 0; i < arraySize; i++) {
```
Comment (oleksii-kapustnyk) [inline src/homework10Part1/WorkingWithNumbers.java:13]: remove print

**Review by oleksii-kapustnyk — APPROVED** [review]

File: src/homework10Part1/WorkingWithNumbers.java (hunk 1 above)
Comment (yalokudoyar69) [reply src/homework10Part1/WorkingWithNumbers.java:13]: Is the problem with the simultaneous use of "return" and "print"?  I used both of them to put the original array into the console and to use the array for further sorting. Considering the task "вивести на консоль масив" after all previous steps, I understood it as all 3 arrays (original and 2 sorted arrays) must be seen in the console.

Comment (oleksii-kapustnyk) [reply src/homework10Part1/WorkingWithNumbers.java:13]: if it is method 'createArray' is should only create. but not print

Comment (yalokudoyar69) [reply src/homework10Part1/WorkingWithNumbers.java:13]: Got it, thank you!

**Pull request merged**